x-pack/plugins/cases/public/components/user_actions/user_actions_list.tsx

```diff
@@ -128,7 +126,7 @@ export const UserActionsList = React.memo(
         }
 
         const userActionBuilder = builder({
-          appId,
+          appId: owner[0],
```

Comment:
why `owner` here vs `appId` from `useApplication`?

Reply:
Because they are being used by the session storage in the comment editor, we want something available on the first render. It is not bullet-proof tbh.